src/components/App.tsx

```diff
@@ -8,6 +10,8 @@ import { PrivateRoute } from './shared/protectedRoute';
 import Landing from './static/landing';
 import { NoComponent } from './static/noComponent';
 
+Mixpanel.init(MIXPANEL_TOKEN_ID);
```

Comment:
Is it possible to initialize mixpanel in index.tsx?

Reply:
inside index.tsx add
```suggestion
Mixpanel.init(process.env.REACT_APP_MIXPANEL_TOKEN_ID!);
```